src/language-usability.rst

```diff
@@ -19,5 +19,133 @@ API Usability
+
+Algorithm Animation
+^^^^^^^^^^^^^^^^^^^
+
+**Algorithm animation** techniques aim to communicate the behavior of algorithms through animated visual notation, sometimes together with natural language commentary. It has been used with success in teaching computer science courses, designing and analyzing algorithms, producing technical drawings, tuning performance, and documenting programs.
+
+.. container:: bib-item
+
+  .. bibliography:: language-usability.bib
+    :filter: key == "brown1991zeus"
+
+ This paper presents a system for algorithm animation, *Zeus*. To a programmer, *Zeus* can be viewed as a general-purpose framework for associating multiple client-defined views with a set of client-defined events, emitted while an algorithm executes. According to the author, constructing animations in *Zeus* appears to be easy and straightforward. Particularly, the concept of Objects make it easy to reuse views, and to build sophisticated views by composing and subclassing other views. However, those claims were not rigorously evaluated in the paper.
+
+.. container:: bib-item
+
+  .. bibliography:: language-usability.bib
+    :filter: key == "stasko1993methodology"
+
+Instead of focusing on how to build an easy-to-use algorithm animation system, this paper discusses why we need different forms of visualization for different classes of algorithms, i.e. **application-specific** visualization, and specifically, what is the requirements for parallel program visualizations. Also, a program animation tool called POLKA is developed for case study. It is claimed that by taking advantage of such application-specific visualizations, programmers could rapidly assess the program's correctness, though without rigorous evaluation.
+
+Program Visualization
+^^^^^^^^^^^^^^^^^^^^^
+
+**Program visualization** is the use of various techniques to help programmers comprehend computer programs. Program visualization techniques take into account various sources of information, including the structure of the source code and run-time behavior. Some program visualizations designed to aid significantly the practice of professional programming and software engineering, while others are focused on aiding the learning of computer science by novice programmers.
+
+.. container:: bib-item
+
+  .. bibliography:: language-usability.bib
+    :filter: key == "maletic2002task"
+
+This paper defines five dimensions to describe several software visualization systems that have very different features along those defined dimensions:
```

Comment:
If this is about software visualization more generally, rather than program visualization, it should be listed in the introduction area.